### PR TITLE
Chore: Add Additional Metadata for Badlion.BadlionClient

### PR DESCRIPTION
**Pull request by Trenly**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19173)

### CODE REVIEW

File: manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.locale.en-US.yaml [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+# CopyrightUrl: 
+ShortDescription: Badlion Client Minecraft Launcher
+Description: Badlion Client is a Minecraft launcher with many added features. It includes anticheat, built-in FPS boost and other mods, and is compatible with most versions of minecraft 1.7 or newer.
+Moniker: BLC
+Tags:
+- minecraft
+- optifine
 ManifestType: defaultLocale
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```
Comment (ItzLevvie) [inline manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.locale.en-US.yaml:24]:
```suggestion
ManifestVersion: 1.0.0

```
This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

File: manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.yaml [hunk 2]
```diff
@@ -4,4 +4,4 @@ PackageIdentifier: Badlion.BadlionClient
 PackageVersion: 3.2.0
 DefaultLocale: en-US
 ManifestType: version
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```
Comment (ItzLevvie) [inline manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.yaml:7]:
```suggestion
ManifestVersion: 1.0.0

```
This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

File: manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.installer.yaml [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+  Scope: user
   InstallerUrl: https://client-updates-cdn77.badlion.net/Badlion%20Client%20Setup%203.2.0.exe
   InstallerSha256: C43392F5C2417BB28F35A19F2DABC8DFB94DA070A647641216625C511F21F3EF
+  # InstallerSwitches:
+  #  Custom: 
+  # ProductCode:
+  UpgradeBehavior: install
 ManifestType: installer
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```
Comment (ItzLevvie) [inline manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.installer.yaml:23]:
```suggestion
ManifestVersion: 1.0.0

```
This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.yaml (hunk 2 above)
Comment (Trenly) [reply manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.yaml]: Thanks for the advice. I wasn't aware that it changed the validation rules

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**